apps/codelab/src/app/codelabs/not-found/not-found.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'codelab-not-found',
+  templateUrl: './not-found.component.html',
+  styleUrls: ['./not-found.component.scss']
```

Comment:
Add `changeDetection: ChangeDetectionStrategy.OnPush`
While there's not much value in it in 404 component, it's nice to have it for consistency.

Reply:
Ok. Done